code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm

```diff
@@ -642,6 +642,37 @@
 		playsound(M, "sparks", 50, 1)
 	..()
 
+/datum/reagent/toxin/rotatium //Rotatium. Fucks up your rotation and is hilarious
+	name = "Rotatium"
+	id = "rotatium"
+	description = "A constantly swirling, oddly colourful fluid. Causes the consumer's sense of direction and hand-eye coordination to become wild."
+	reagent_state = LIQUID
+	color = "#FFFF00" //RGB: 255, 255, 0 Bright ass yellow
+	metabolization_rate = 0.6 * REAGENTS_METABOLISM
+	toxpwr = 0
+	var/rotate_timer = 0
+	var/randrot = 1
+	var/hold = 0
+
+/datum/reagent/toxin/rotatium/on_mob_life(mob/living/M)
+	rotate_timer++
+	if(M.reagents.get_reagent_amount("rotatium") < 2)
+		M.client.dir = 1
+		return
+	if(rotate_timer >= rand(5,30)) //Random rotations are wildly unpredictable and hilarious
+		rotate_timer = 0
+		hold = rand(1,8)
+		switch(hold)
```

Comment:
so it's a 1 in 4 to pick 1 from a list of 4...

why is this not ```M.client.dir = pick(2,4,6,8)```

also 6 is SOUTHEAST, and I dont think that actually works for ```client.dir```